pkg/block/local/adapter_test.go

```diff
@@ -1,231 +1,24 @@
 package local_test
 
 import (
-	"context"
-	"io"
-	"os"
-	"path/filepath"
-	"sort"
-	"strings"
+	"path"
 	"testing"
 
-	"github.com/go-test/deep"
 	"github.com/treeverse/lakefs/pkg/block"
+	"github.com/treeverse/lakefs/pkg/block/blocktest"
 	"github.com/treeverse/lakefs/pkg/block/local"
-	adapterTest "github.com/treeverse/lakefs/pkg/block/test"
-	"github.com/treeverse/lakefs/pkg/testutil"
 )
 
 const testStorageNamespace = "local://test"
 
 func TestLocalAdapter(t *testing.T) {
-	var params []func(*local.Adapter)
-	adapterTest.TestAdapter(t, block.BlockstoreTypeLocal, params)
-}
-
-func makeAdapter(t *testing.T) *local.Adapter {
-	t.Helper()
-	dir, err := os.MkdirTemp("", "testing-local-adapter-*")
-	testutil.MustDo(t, "TempDir", err)
-	testutil.MustDo(t, "NewAdapter", os.MkdirAll(dir, 0700))
-	a, err := local.NewAdapter(dir)
-	testutil.MustDo(t, "NewAdapter", err)
-
-	t.Cleanup(func() {
-		if _, ok := os.LookupEnv("GOTEST_KEEP_LOCAL"); ok {
-			return
-		}
-		_ = os.RemoveAll(dir)
-	})
-	return a
-}
-
-func makePointer(path string) block.ObjectPointer {
-	return block.ObjectPointer{Identifier: path, StorageNamespace: testStorageNamespace}
-}
-
-func TestLocalPutExistsGet(t *testing.T) {
```

Comment:
Just making sure we have the same coverage with the contract tests, consider keeping these for a while until we do.

Reply:
The tests were migrated